source/_integrations/vesync.markdown

```diff
@@ -138,7 +138,8 @@ Sensors and settings exposed by VeSync humidifiers.
 
 | Number                  | Description                                                                        | Example   |
 | ----------------------- | ---------------------------------------------------------------------------------- | --------- |
-| `mist_level`            | Mist level intensity (Range: 1-9, Step: 1)                                         | 1         |
+| `mist_level`            | Mist level intensity (Range: 1-9, Step: 1). Only available in manual mode.         | 1         |
+| `night_light_brightness`| Night light brightness level (Values: 0, 50, 100).                                 | 0         |
```

Comment:
Do the docs here need to be updated? Asking because in core I see "bright, dim, off".